components/Tabs/src/MDCTabBarIndicatorAttributes.m

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTabBarIndicatorAttributes.h"
+
+@implementation MDCTabBarIndicatorAttributes
+
+#pragma mark - NSCopying
+
+- (instancetype)copyWithZone:(__unused NSZone *)zone {
+  MDCTabBarIndicatorAttributes *attributes = [[[self class] alloc] init];
+  attributes.path = _path;
+  return attributes;
+}
+
+#pragma mark - NSObject
+
+- (NSString *)description {
+  return [NSString stringWithFormat:@"%@ path:%@", [super description], _path];
+}
+
+- (BOOL)isEqual:(id)object {
+  if (![object isKindOfClass:[self class]]) {
+    return NO;
+  }
+
+  MDCTabBarIndicatorAttributes *otherAttributes = object;
```

Comment:
nit: Add explicit cast?

Reply:
LGTM.
Sorry about that, old habits from a team that prohibited implicit casting.